build.gradle

```diff
@@ -1,24 +1,24 @@
 buildscript {
     repositories {
+        mavenLocal()
         maven {
             url "https://raw.githubusercontent.com/SciAps/nexus/master/"
         }
         mavenCentral()
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:3.5.0'
     }
 }
 
 apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 22
-    buildToolsVersion '27.0.3'
 
     defaultConfig {
-        minSdkVersion 22
+        minSdkVersion 17
```

Comment:
I guess reducing the minSdkVersion is less restrictive, but just curious to know the reasoning behind this.

Reply:
@mbalasciaps I am going to include this project to XRFAndroid. So, the `minSdkVersion 17` value is typical there.